openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/7-csi-functional/XYNP-Remote-Restore-Diff-ns/remote-restore-diff-ns

```diff
@@ -0,0 +1,124 @@
+}
+
+node() {
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+releaseTag=$(echo $4)
+velero_version=$(echo $5)
+velero_plugin_version=$(echo $6)
+sdk_token=$(echo $7)
+case_id=XYNP
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash openebs-konvoy-e2e/utils/e2e-cr jobname:backup-restore-cstor-csi-diff-ns jobphase:Waiting
+bash openebs-konvoy-e2e/utils/pooling jobname:backup-restore-cstor-csi
+bash openebs-konvoy-e2e/utils/e2e-cr jobname:backup-restore-cstor-csi-diff-ns jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
+
+################
+# LitmusBook 1 #
+################
+
+echo "*******Deploying Percona Application using CSI****"
+
+run_id="diff-ns";test_name=$(bash openebs-konvoy-e2e/utils/generate_test_name testcase=percona-deployment metadata=${run_id})
+echo $test_name
+
+cd litmus
+echo "Running the litmus test for Percona Deployment.."
+
+# Storeing GCP service account key in file"
+#echo "$sdk_token" > key.json
+
+#echo "Creating configmap in litmus namespace for GCP token"
+#kubectl create cm gcp-key -n litmus --from-file /var/sdk/key.json
+
+cp apps/percona/deployers/run_litmus_test.yml percona_backup_restore_csi_diff_ns.yml
+
+sed -i -e 's/app: percona-deployment/app: deploy-percona-backup-restore-csi-diff-ns/g' \
+-e 's/value: openebs-standard/value: openebs-cstor-csi/g' \
+-e 's/value: app-percona-ns/value: percona-backup-csi-diff-ns/g' percona_backup_restore_csi_diff_ns.yml
+
+sed -i '/command:/i \
+          - name: RUN_ID\
+            value: '"$run_id"'\
+' percona_backup_restore_csi_diff_ns.yml
+
+cat percona_backup_restore_csi_diff_ns.yml
+
+bash ../openebs-konvoy-e2e/utils/litmus_job_runner label='app:deploy-percona-backup-restore-csi-diff-ns' job=percona_backup_restore_csi_diff_ns.yml
+cd ..
+bash openebs-konvoy-e2e/utils/dump_cluster_state;
+bash openebs-konvoy-e2e/utils/event_updater jobname:backup-restore-cstor-csi-diff-ns $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+if [ "$?" != "0" ]; then
+python3 openebs-konvoy-e2e/utils/result/result_update.py $job_id XYNP 3-functional "Create backup and restore and verify if the application is not impacted" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+################
+# LitmusBook 2 #
+################
+
+echo "*******Performing Backup and Restore on Target**********"
+
+test_name=$(bash openebs-konvoy-e2e/utils/generate_test_name testcase=velero-backup-restore metadata="")
+echo $test_name
+
```

Comment:
@shashank855  here also append with run_id? Wherever using the same testcase name in alll the places can you append